packages/api/cms-api/src/blocks/createSeoBlock.ts

```diff
@@ -162,5 +162,5 @@ export function createSeoBlock<ImageBlock extends Block = typeof PixelImageBlock
         }
     }
 
-    return createBlock(SeoBlockData, SeoBlockInput, "Seo");
+    return createBlock(SeoBlockData, SeoBlockInput, "SEO");
```

Comment:
Changing the name of this block is an incompatible change, as the generated block types will be different: `SeoBlockData` -> `SEOBlockData`. This will cause type errors in all frontends.

I don't think we should change this name at all. It's just used in code and not as a label. And it is inconsistent with `SeoBlockData` and `SeoBlockInput`

Reply:
you are right, I've changed that.